.github/workflows/submitty_ci.yml

```diff
@@ -15,6 +15,10 @@ env:
   NODE_VERSION: 16
   PYTHON_VERSION: 3.8
 
+concurrency:
+  # Cancel running jobs for previous commits to any branch except for main
+  group: ${{ (github.ref && github.ref_name != 'main') || github.run_id }}
```

Comment:
> Otherwise, it will use the ref name as the identifier so that adding a commit to a branch/pull and re-running the workflow will cancel the previous run on the branch/pull.

That's the intent.  We don't want to cancel jobs associated with `main`, but we want to cancel everything else if new commits are added.

Reply:
We only want to cancel jobs for previous commits on a given branch

Reply:
Oh yeah, right.  I'll try your changes.

Reply:
There still seems to be an issue where commits to `main` will still be cancelled.  I'll investigate further tomorrow.